app/celer-sim/RunnerInputIO.json.cc

```diff
@@ -157,6 +158,8 @@ void to_json(nlohmann::json& j, RunnerInput const& v)
     LDIO_SAVE(geometry_file);
     LDIO_SAVE(physics_file);
     LDIO_SAVE_OPTION(event_file);
+    LDIO_SAVE_WHEN(num_clumped_events,
+                   !v.event_file.empty() && ends_with(v.event_file, ".root"));
```

Comment:
The `ends_with` function knows that if the string isn't long enough for the suffix, the result is "no" rather than "segfault" 😄 
```suggestion
                   ends_with(v.event_file, ".root"));
```